crates/sozo/src/commands/execute.rs

```diff
@@ -12,7 +12,7 @@ use crate::ops::execute;
 #[command(about = "Execute a system with the given calldata.")]
 pub struct ExecuteArgs {
     #[arg(help = "The address of the contract to be executed.")]
-    pub contract: FieldElement,
+    pub contract_name: String,
```

Comment:
we should support either here, so if it is a `0x` treat it as an address, else as a name